docs/status.rst

```diff
@@ -0,0 +1,83 @@
+Status Report
+=============
+
+.. note:: This page will be updated as development progresses. If you have any questions or something appears out of date please feel free to :doc:`contact us <support>`.
+
+Development wraps around the three pillars identified with the help of CERN physicists in the `5th CAP meeting <https://github.com/cernanalysispreservation/analysis-preservation.cern.ch/wiki/Fifth-CAP-meeting>`_:
+
+- `Describe`_
+- `Capture`_
+- `Rerun`_
+
+To this point these can roughly be translated to *what you see*, *the functionality that lies behind it* and *additional services*.
+
+Describe
+--------
+
+Experiment schemas are used to describe the most important features and data for an analysis. They will be accessible via web forms and an API. Thus, this pillar includes everything related to the *schemas*, *search*, *user interface* and *API*.
+
+**Schema - work in progress**
+	**ALICE - information required**
+		Pending, as no schema or information for creating a schema has been provided yet.
+	**ATLAS - untested**
+		Focus has been put on the workflow. A basic schema exists that will require testing and feedback.
+	**CMS - iterative testing and development**
+		Iterative testing has provided us with sufficient input to create a schema that covers basic analyses. More feedback is needed from groups that have further requirements.
+		
+		The CMS Questionnaire has been added and will undergo testing to ensure its quality.
+	**LHCb - polishing state**
+		Testing and refinement have let to an (almost) complete schema that will undergo full-functionality testing to ensure its quality.
+**Search - being implemented**
+	A dedicated search page is currently under development. We have gained some feedback pointing out interesting filters for adjusting the search to specific analysis information. More feedback is welcome.
+**UI - in progress of updating**
+	An update for the user interface is work in progress and will continue for some while. We will need testers to ensure the best user experience.
+**API - basic version exists**
+	A very basic version exists, future development will be driven by requirements named by physicists. This is one of the next tasks to come.
+
+Capture
```

Comment:
The capture status cannot be exposed right now as we are awaiting some strategic approvals during the next weeks. Here we are touching internals so it becomes tricky. If possible, I would like to postpone the exposure of this part by a few weeks - at least the details